packages/lerna/src/commands/add-caching/index.ts

```diff
@@ -108,29 +126,7 @@ class AddCachingCommand extends Command {
     );
   }
 
-  convertAnswersToNxConfig(answers: UserAnswers) {
-    const nxJsonPath = joinPathFragments(workspaceRoot, "nx.json");
-    let nxJson: NxJsonConfiguration = {};
-    try {
-      nxJson = readJsonFile(nxJsonPath);
-      // eslint-disable-next-line no-empty
-    } catch {}
-
-    nxJson.tasksRunnerOptions = nxJson.tasksRunnerOptions || {};
-    nxJson.tasksRunnerOptions["default"] = nxJson.tasksRunnerOptions["default"] || {};
-    nxJson.tasksRunnerOptions["default"].runner =
-      nxJson.tasksRunnerOptions["default"].runner || "nx/tasks-runners/default";
-    nxJson.tasksRunnerOptions["default"].options = nxJson.tasksRunnerOptions["default"].options || {};
-
-    if (nxJson.tasksRunnerOptions["default"].options.cacheableOperations) {
-      this.logger.warn(
-        "add-caching",
-        "The `tasksRunnerOptions.default.cacheableOperations` property already exists in `nx.json` and will be overwritten by your answers"
-      );
-    }
```

Comment:
Adding the default task runner seems like excess config. Nx should default this for us when this is missing.

The meaningful removal here is of `cacheableOperations`, which moved to `targetDefaults`.